### ISSUE TITLE
Dry run

### ISSUE THREAD

**Issue by threaz** [Oefenweb/ansible-percona-server#34]
Hi. When I'm trying to run playbook containing `oefenweb.percona-server` role with `--check` option  I get the following error
```
TASK [oefenweb.percona-server : configure | service | stop] **********************************************************************************************************************************************************************************************************************
fatal: [db1.staging]: FAILED! => {"msg": "The conditional check '_mysqld_installed.rc != 0' failed. The error was: error while evaluating conditional (_mysqld_installed.rc != 0): 'dict object' has no attribute 'rc'\n\nThe error appears to have been in '/home/damian/Projects/auth/ansible/.galaxy_roles/oefenweb.percona-server/tasks/configure.yml': line 16, column 5, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n- block:\n  - name: configure | service | stop\n    ^ here\n"}
```

It looks like `_mysqld_installed` is not registered during dry-run. Is there a way I can achieve this so that I can safely review changes that will be done to my database?

EDIT:
I ran my playbook with `-v` and `_mysqld_installed` is not set indeed because `first-run.yml` is `skipped`.
```
TASK [oefenweb.percona-server : first-run | check] *******************************************************************************************************************************************************************************************************************************
skipping: [db1.staging] => {"changed": false, "msg": "remote module (command) does not support check mode"}
```
I managed to make it work by adding `check_mode: no` (https://docs.ansible.com/ansible/latest/user_guide/playbooks_checkmode.html#enabling-or-disabling-check-mode-for-tasks) in `first-run.yml` though.

**Comment by tersmitten** [comment]
Can you make a pull request?

**Comment by threaz** [comment]
I haven't done it because the solution I suggested was applied a few days after my comment https://github.com/Oefenweb/ansible-percona-server/pull/35/files :)

**Comment by tersmitten** [comment]
Fixed in #35
